Ready to merge: Add line diagram

panmona opened the pull request:
- Add the tile for the `market`
- fix order of depot 
- add padding to github link in footer
- remove links in header
- move areaChart to common components
- change yAxis of capitalDevelopment

closes #56 and #32

panmona committed:
Merge branch 'master' into add-line-diagram

RohrerF commented:
We could take a look at [this](https://github.com/bvaughn/react-virtualized) to speed up the performance by only rendering the visible charts

panmona committed:
disable animation

RohrerF commented:
@maracuja-juice  To fix the Labels of the diagramm we could try to save the string  of the label to the store. The Labelproblem exists in the CapitalChartDiagramm

panmona committed:
resolve conflicts

panmona committed:
work on the input button

doesn't seem to fully work yet. the snapshot updates make it more difficult to test

panmona commented:
@RohrerF How can i pause the generation of new snapshots?

panmona committed:
make input field smaller, add arrow

panmona committed:
remove todos

panmona committed:
update yarn lock with new one from master

RohrerF committed:
Merge branch 'master' into add-line-diagram

panmona committed:
fix react typing version

panmona committed:
change stock config

panmona committed:
add arrow

panmona committed:
implement category icon

panmona committed:
remove build folder

panmona committed:
switch to a switch instead of ternary operator

panmona committed:
move stock area chart to components and move components in depot

panmona committed:
extract components from stockCard and change stockType to union type

panmona committed:
change footer padding

panmona committed:
remove links from header

panmona committed:
change capital config to last 10 minutes

panmona committed:
update snapshots

panmona committed:
remove conditional show logic

panmona committed:
rename area chart to financialdevelopmentchart

panmona committed:
change capital config to 20 minutes
